cms/templates/js/course_grade_policy.underscore

```diff
@@ -2,31 +2,31 @@
   <div class="field text" id="field-course-grading-assignment-name">
   	<label for="course-grading-assignment-name"><%= gettext("Assignment Type Name") %></label>
   	<input type="text" class="long" id="course-grading-assignment-name" value="<%= model.get('type') %>" />
-  	<span class="tip tip-stacked"><%= gettext("e.g. Homework, Midterm Exams") %></span>
+  	<span class="tip tip-stacked"><%= gettext("The general category for this type of assignment, such as Homework or Midterm Exams. This name is visible to learners.") %></span>
```

Comment:
Only that it seems plainer to me. I don't feel strongly about it. I think
"for example" matches a string a little lower in the file though.

On Fri, Mar 25, 2016 at 3:00 PM, Carol Tong <notifications@github.com>
wrote:

> In cms/templates/js/course_grade_policy.underscore
> <https://github.com/edx/edx-platform/pull/11944#discussion_r57477832>:
>
> > @@ -2,31 +2,31 @@
> >    <div class="field text" id="field-course-grading-assignment-name">
> >    	<label for="course-grading-assignment-name"><%= gettext("Assignment Type Name") %></label>
> >    	<input type="text" class="long" id="course-grading-assignment-name" value="<%= model.get('type') %>" />
> > -  	<span class="tip tip-stacked"><%= gettext("e.g. Homework, Midterm Exams") %></span>
> > +  	<span class="tip tip-stacked"><%= gettext("The general category for this type of assignment, such as Homework or Midterm Exams. This name is visible to learners.") %></span>
>
> Thanks @pdesjardins <https://github.com/pdesjardins> Is there a reason
> why "for example" is preferred over "such as"?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/11944/files/d9d2d5a0584bf24b9f1403b8542a0341ed83a4d8#r57477832>
>